src/views/Home/components/SemesterEvents/SemesterEvents.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../../vars';
```

Comment:
Is this just a copy of other code?  If so, then we should use that code instead of copying it.